<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxGeneral_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>337</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Copyright</string>
       </property>
       <widget class="QLabel" name="copyrightLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>271</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;b&gt;HodgkinHuxley Simulator:&lt;/b&gt;&lt;br&gt;
Copyright 2013 by Robin Rump&lt;br&gt;
&lt;a href=&quot;http://www.qtcustomplot.com/&quot;&gt;http://www.robinrump.com/&lt;/a&gt;&lt;br&gt;&lt;br&gt;
&lt;b&gt;QCustomPlot Library:&lt;/b&gt;&lt;br&gt;
Copyright 2011, 2012, 2013 by &lt;br&gt;
Emanuel Eichhammer&lt;br&gt;
&lt;a href=&quot;http://www.qtcustomplot.com/&quot;&gt;http://www.qtcustomplot.com/&lt;/a&gt;&lt;br&gt;&lt;br&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSliders">
      <attribute name="title">
       <string>Sliders</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxCurrent">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>337</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Current</string>
       </property>
       <widget class="QSpinBox" name="minCurrentValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>27</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-500</number>
        </property>
        <property name="maximum">
         <number>499</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="maxCurrentValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>57</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-499</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QLabel" name="maxCurrentLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>59</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum current:</string>
        </property>
       </widget>
       <widget class="QLabel" name="minCurrentLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>29</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum current:</string>
        </property>
       </widget>
       <widget class="QLabel" name="minCurrentUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>30</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mA</string>
        </property>
       </widget>
       <widget class="QLabel" name="maxCurrentUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>30</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mA</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBoxConductivities">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>337</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Conductivites</string>
       </property>
       <widget class="QSpinBox" name="minGNaValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>27</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>499</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="maxGNaValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>57</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QLabel" name="maxGNaLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>59</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum gNa:</string>
        </property>
       </widget>
       <widget class="QLabel" name="minGNaLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>29</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum gNa:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="minGKValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>88</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>499</number>
        </property>
       </widget>
       <widget class="QLabel" name="minGKLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum gK:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="maxGKValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>118</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QLabel" name="maxGKLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum gK:</string>
        </property>
       </widget>
       <widget class="QLabel" name="minGLLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>152</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum gL:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="minGLValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>499</number>
        </property>
       </widget>
       <widget class="QLabel" name="maxGLLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>182</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum gL:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="maxGLValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>180</y>
          <width>57</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
       <widget class="QLabel" name="minGNaUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mS/cm2</string>
        </property>
       </widget>
       <widget class="QLabel" name="maxGNaUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mS/cm2</string>
        </property>
       </widget>
       <widget class="QLabel" name="minGKUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>90</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mS/cm2</string>
        </property>
       </widget>
       <widget class="QLabel" name="maxGKUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mS/cm2</string>
        </property>
       </widget>
       <widget class="QLabel" name="minGLUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>150</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mS/cm2</string>
        </property>
       </widget>
       <widget class="QLabel" name="maxGLUnit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>mS/cm2</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabAppearances">
      <attribute name="title">
       <string>Appearances</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxAppearances">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>337</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Colors</string>
       </property>
       <widget class="QLabel" name="colorCurrentLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>59</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Color current:</string>
        </property>
       </widget>
       <widget class="QLabel" name="colorMembraneLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>29</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Color membrane:</string>
        </property>
       </widget>
       <widget class="QLabel" name="colorNLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Color n:</string>
        </property>
       </widget>
       <widget class="QLabel" name="colorMLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Color m:</string>
        </property>
       </widget>
       <widget class="QLabel" name="colorHLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>152</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Color h:</string>
        </property>
       </widget>
       <widget class="QToolButton" name="colorMembraneButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>29</y>
          <width>27</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="colorMembraneValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>29</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="colorCurrentButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>27</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="colorCurrentValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="colorNButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>27</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="colorNValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="colorMButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>120</y>
          <width>27</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="colorMValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>120</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="colorHButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>27</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="colorHValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
